main.go

```diff
@@ -160,6 +165,6 @@ func interruptHandler(l net.Listener) {
 	notifier := make(chan os.Signal, 1)
 	signal.Notify(notifier, os.Interrupt, syscall.SIGTERM)
 	<-notifier
-	log.Info("Received SIGINT/SIGTERM; exiting gracefully...")
+	// level.Info(logger).Log("msg", "Received SIGINT/SIGTERM; exiting gracefully...")
```

Comment:
Why is this commented now?

Reply:
This is commented out because I need to figure out how to pass the `logger` instance in there.

Reply:
So you said you have a problem with this one. What's the nature of the problem?

Reply:
logger is defined within `func main()` (not global) so `func interruptHandler` can't see it. I either need to move logger as a global singleton (probably not wise) or refactor interuptHandler to get a reference to logger

Reply:
I see. Yeah, I guess giving the `func interruptHandler` an additional argument for the logger is the way to go.